node-src/lib/getDependencies.test.ts

```diff
@@ -22,7 +22,7 @@ describe('getDependencies', () => {
     const dependencyNames = Array.from(dependencies).map((dependency) => dependency.split('@@')[0]);
     expect(dependencyNames).toEqual(
       expect.arrayContaining([
-        ...Object.keys(packageJson.dependencies),
+        ...Object.keys(packageJson.dependencies || {}),
```

Comment:
This is an antipattern in tests. The expect should be much more concrete about what is expects based on a set of inputs.

Reply:
@thafryer this is required because we don't have any dependencies in our package.json.
The test fails without an empty object. Yarn Berry removes `dependencies: {}` from the `package.json` if no dependencies exist.

Reply:
So, if an empty object is expected for this specific case, why not just be explicit and use the empty object?

Reply:
Great callout here. We need to change the target packageJson to something that has dependencies to mimic a real use case.